src/handlers/user.rs

```diff
@@ -142,13 +142,9 @@ pub async fn add_user(
     );
     user.save(&appstate.pool).await?;
 
-    // add LDAP user
-    // FIXME: check for LDAP being enabled
-    if true {
```

Comment:
czemu nie zaimplementować tego warunku?

Reply:
Moim zdaniem to powinno działać analogicznie do wysyłania maili , czyli handlerów to nie powinno obchodzić